python/lsst/summit/utils/astrometry/plotting.py

```diff
@@ -59,14 +60,14 @@ def plot(exp, icSrc=None, filteredSources=None, saveAs=None,
         Smooth the image slightly to improve the visability of low SNR sources?
     """
     fig = plt.figure(figsize=(16, 16))
-    arr = exp.image.array
-    arr = np.clip(arr, clipMin, clipMax)
+    data = copy.deepcopy(exp.image.array)
+    data = np.clip(data, clipMin, clipMax)
     if doSmooth:
-        arr = quickSmooth(arr)  # smooth slightly to help visualize
-    plt.imshow(np.arcsinh(arr)/10,
+        data = quickSmooth(data)  # smooth slightly to help visualize
```

Comment:
If this is acting in-place (as the commit implies), maybe it'd be better not to do anything with the return value, just in terms of readability.

Reply:
Thanks.